fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/DefaultLoanLifecycleStateMachine.java

```diff
@@ -126,6 +126,11 @@ private LoanStatus getNextState(LoanEvent loanEvent, Loan loan) {
                     newState = overpaidTransition();
                 }
             break;
+            case LOAN_CHARGEBACK:
+                if (anyOfAllowedWhenComingFrom(from, LoanStatus.CLOSED_OBLIGATIONS_MET, LoanStatus.ACTIVE)) {
+                    newState = activeTransition();
```

Comment:
This one is not complete:

If it was ACTIVE
-  it remains active,
if it was CLOSED_OBLIGATIONS_MET 
- it goes to OVERPAID
if it was OVERPAID (missing from the list)
- it remains OVERPAID

(i double check the last option)